spec/02-sotd.md

```diff
@@ -0,0 +1,3 @@
+This is the First Public Working Draft
+([FPWD](https://www.w3.org/2017/Process-20170301/#working-draft)) of a MQTT
+trace context protocol specification.
```

Comment:
I believe that "MQTT Trace Context" is the proper name of the spec, no? If so, it should have title case.

```suggestion
Trace Context specification.
```